### PR TITLE
SDL_DISABLE_WINDOWS_IME

### PR DESCRIPTION
**Pull request by uhuntu**
https://github.com/Rust-SDL2/rust-sdl2/issues/1041

### CODE REVIEW

**Comment by Biswa96** [comment]
1. Run `updpkgsums` to update the checksums.
2. Increase `pkgrel` value by 1.

**Comment by uhuntu** [comment]
Thank you @Biswa96

File: mingw-w64-SDL2/PKGBUILD [hunk 1]
```diff
@@ -40,6 +42,7 @@ prepare() {
   patch -p1 -i ${srcdir}/003-fix-static-library-name.patch
   patch -p1 -i ${srcdir}/004-dont-duplicate-libs-in-libs-private.patch
   patch -p1 -i ${srcdir}/005-sdl-config-win-paths.patch
+  patch -p1 -i ${srcdir}/SDL2-2.0.12-SDL_DISABLE_WINDOWS_IME.patch
```
Comment (lazka) [inline mingw-w64-SDL2/PKGBUILD:45]: please add some comments like you did here: https://github.com/Rust-SDL2/rust-sdl2/pull/1042/files

Comment (uhuntu) [reply mingw-w64-SDL2/PKGBUILD:47]: As your wish, https://github.com/msys2/MINGW-packages/pull/7364/commits/8521c5fb7b345aa9c798dc1ab020cebad1e69012

**Pull request merged**